modules/lambda@edge/package.tf

```diff
@@ -0,0 +1,10 @@
+data "archive_file" "lambda_zip" {
+  for_each = local.functions
+
+  type             = "zip"
+  source_dir       = each.value.local_path
+  output_file_mode = "0666"
+  output_path      = "${path.module}/archives/${each.key}.zip"
```

Comment:
Can we also support inline files?

Reply:
Sure, i'll look into that. I don't think there's a reason we can't.

Reply:
I've updated the schema.